optaplanner-examples/src/test/java/org/optaplanner/examples/common/app/SolverPerformanceTest.java

```diff
@@ -43,13 +51,40 @@
  * Always use a {@link Test#timeout()} on {@link Test}, preferably 10 minutes because some of the Jenkins machines are old.
  * @param <Solution_> the solution type, the class with the {@link PlanningSolution} annotation
  */
+@RunWith(Parameterized.class)
 public abstract class SolverPerformanceTest<Solution_> extends LoggingTest {
 
-    private static final String MOVE_THREAD_COUNT_OVERRIDE = System.getProperty(TestSystemProperties.MOVE_THREAD_COUNT);
+    private static final String MOVE_THREAD_COUNTS_VALUE = System.getProperty(TestSystemProperties.MOVE_THREAD_COUNTS);
+    private static final String DEFAULT_MOVE_THREAD_COUNT = "NONE";
 
     protected SolutionFileIO<Solution_> solutionFileIO;
     protected String solverConfig;
 
+    private String moveThreadCount;
```

Comment:
I'll make this final and move it up a bit because variables are usually in the order of non-volatile to volatile

Reply:
+1